src/client.ts

```diff
@@ -79,6 +88,14 @@ export class DgraphClient {
     }
 }
 
+// isJwtExpired returns true if the error indicates that the jwt has expired.
+export function isJwtExpired(err: any): Boolean { // tslint:disable-line no-any
```

Comment:
It is because in `try...catch(err)` block, `err` of type `any`.

I had tried mapping it to `Error` interface to `catch` block, but got the error.
```typescript
Catch clause variable cannot have a type annotation.ts(1196)
```